lib/generators/hyrax/work_resource/work_resource_generator.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails/generators'
+require 'rails/generators/model_helpers'
+
+class Hyrax::WorkResourceGenerator < Rails::Generators::NamedBase
+  # ActiveSupport can interpret models as plural which causes
+  # counter-intuitive route paths. Pull in ModelHelpers from
+  # Rails which warns users about pluralization when generating
+  # new models or scaffolds.
+  include Rails::Generators::ModelHelpers
+
+  source_root File.expand_path('../templates', __FILE__)
```

Comment:
Kind of a meh, but: could also do `File.expand_path('templates', __dir__)` ?

Reply:
i think this line is just ripped from the old work generator. i'm open to whatever refactors here

Reply:
k, consistency is more important then IMO. `wontfix`